src/link/link.spec.ts

```diff
@@ -16,6 +16,12 @@ class TestLinkComponent {
 class TestDisabledLinkComponent {
 }
 
+@Component({
+	template: `<a href="https://angular.carbondesignsystem.com/" [inline]="1+1===2" ibmLink>link</a>`
```

Comment:
Not sure why we need the `1+1===2` there? I would recommend just passing `true`
```suggestion
	template: `<a href="https://angular.carbondesignsystem.com/" [inline]="true" ibmLink>link</a>`
```

Reply:
Was mostly just following the test structure already in place for disabled links! Would normally do this otherwise! Good to know for future reference though.

Reply:
Ah, not sure how that got like that 😮